File: Java/3 - Starbuzz/src/main/java/com/mycompany/starbuzz/Espresso.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.starbuzz;

import static com.mycompany.starbuzz.Beverage.Size.GRANDE;
import static com.mycompany.starbuzz.Beverage.Size.TALL;
import static com.mycompany.starbuzz.Beverage.Size.VENTI;

/**
 *
 * @author ismas
 */
public class Espresso extends Beverage{
    public Espresso(){
        description = "Espresso";
    }
    
    @Override
    public double cost(){
        switch(this.getSize()){
            case TALL:
                return 1.99;
            case GRANDE:
                return 2.49;
            case VENTI:
                return 2.89;
            default:
                return 1.99;
        }
    }
}

File: C#/3. Starbuzz (Decorator)/Classes/Espresso.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Starbuzz.Classes
{
    public class Espresso : Beverage
    {
        public Espresso()
        {
            _Description = "Espresso";
        }

        public override double Cost()
        {
            switch (_Size)
            {
                case Size.TALL:
                    return 1.99f;
                case Size.GRANDE:
                    return 2.49f;
                case Size.VENTI:
                    return 2.89;
                default:
                    return 1.99f;
            }
        }
    }
}
